README.md

```diff
@@ -1,2 +1,16 @@
-# desktop
-Noop Desktop
+# Noop Desktop
+
+This repo contains the release artifacts for Noop Desktop, a platform for running applications locally on Workshop and in Noop Cloud.
+
+Noop Desktop includes Cloud Console (the management console for Applications running in the cloud) and Workshop (a local instance of all Cloud functionality).
+
+We currently release binaries for Apple aarch64 and Intel x86_64 chips as well as Linux x86_64. See [releases](https://github.com/noop-inc/desktop-lima/releases) for current version and release notes.
```

Comment:
```suggestion
We currently release binaries for Apple silicon Machines. See [releases](https://github.com/noop-inc/desktop/releases) for current version and release notes.
```

Corrected repo linked. Also removed mention of forthcoming support for Intel Macs and Linux machines (we do not publish binaries for those systems yet).